src/SmoFacade/Database.cs

```diff
@@ -80,6 +93,7 @@ public List<BackupMetadata> RecentBackups()
       dbName.Value = _database.Name;
       cmd.Parameters.Add(dbName);
 
+      cmd.Connection.Open();

```

Comment:
I had issues here as it said "SQLConnection was not open before executing a query".
Is this because it should already be opened on line 23?

Reply:
Now getting an error when creating via the `AgDatabase` class - saying "SQLConnection is already open"

Wondering if we should open the connection for the `Database` class in it's constructor?

Reply:
recent commit `591f14d` solves this issue - but want to know your thoughts on it.